pkgs/applications/science/medicine/xmedcon/default.nix

```diff
@@ -31,6 +31,6 @@ stdenv.mkDerivation rec {
     homepage = "https://xmedcon.sourceforge.io/Main/HomePage";
     license = licenses.lgpl2Plus;
     maintainers = with maintainers; [ arianvp flokli ];
-    platforms = with platforms; [ darwin linux ];
+    platforms = platforms.darwin ++ platforms.linux;
```

Comment:
This should be unix.

Reply:
Why? `platforms.unix` includes `cygwin`, `netbsd`, `openbsd` and `redox`, and I doubt we can guarantee this works there.

Reply:
platforms.meta is mostly for hydra and for the bigger platforms. Also if we ever get any better cygwin, netbsd, openbsd or redox support it will be a pain adding the platform to 10s of thousands of packages.

Reply:
IMHO, `platforms.meta` is the field for maintainers to describe what platforms they intend to support, or where it has (once) worked.

If there's enough demand to start supporting another platform, I'm fairly confident there will be plenty of tooling to automatically update these fields to include the new architecture, and way more timeconsuming to bikeshed about this on every PR.